org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.html

```diff
@@ -327,6 +327,98 @@
     </winery-modal-footer>
 </winery-modal>
 
+<winery-modal *ngIf="!readonly" bsModal #threatModelingModal="bs-modal" [modalRef]="threatModelingModal">
+    <winery-modal-header [title]="'Threat Model'">
+    </winery-modal-header>
+    <winery-modal-body>
+        <div id="accordion">
+            <div class="card">
+                <div class="card-header" id="headingOne">
+                    <h5 class="mb-0">
+                        <button class="btn btn-link"  (click)="this.threatModalTab='create'">Create New Threat</button>
+                    </h5>
+                </div>
+
+                <div id="collapseOne" class="collapse" [class.show]="this.threatModalTab==='create'" >
+                    <div class="card-body">
+                        <form (ngSubmit)="this.createNewThreat()">
+                            <div class="form-group row">
+                                <label for="email" class="col-sm-4 col-form-label">Threat name</label>
+                                <div class="col-sm-8">
+                                    <input type="email" class="form-control" id="name" name="name" required [(ngModel)]="this.threatModelingData.threatCreation.name">
```

Comment:
Why do you have the type set to `email`?

Reply:
fixed, used "text" instead